nodebuilder/node/admin.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/celestiaorg/celestia-node/libs/authtoken"
 )
 
-const APIVersion = "v0.2.1"
+const APIVersion = "v0.2.2"
```

Comment:
This needs to match our node version btw

cc @distractedm1nd @MSevey @adlerjohn @Wondertan

Reply:
what do you mean by this? API version is not the same as node version